httplib.h

```diff
@@ -4724,12 +4727,10 @@ get_range_offset_and_length(const Request &req, const Response &res,
 }
 
 inline bool expect_content(const Request &req) {
-  if (req.method == "POST" || req.method == "PUT" || req.method == "PATCH" ||
-      req.method == "PRI" || req.method == "DELETE") {
-    return true;
-  }
   // TODO: check if Content-Length is set
-  return false;
+  return (req.method == "POST" || req.method == "PUT" ||
```

Comment:
This is incorrect. I intentionally put the if statement before the TODO comment.

Reply:
Ah, I see. Something will come after the first check. I will revert this part.